drivers/portalmem/portalmem.c

```diff
@@ -456,18 +460,14 @@ int portalmem_dmabuffer_create(PortalAlloc portalAlloc)
 #endif
                                 sg_dma_address(sg) = sg_phys(sg);
                         }
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(4,1,0))
```

Comment:
this does not look right to me export_info was there and then went away and then came back? Or did RedHat/CentOS back port dmabuf? If that is the case, maybe we can find a better conditional.